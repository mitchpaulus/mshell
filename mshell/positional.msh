[echo "Number 1: " $1 +];
[echo "Number 2: " $2 +];
