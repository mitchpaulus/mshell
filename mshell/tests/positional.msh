#!/usr/bin/env mshell
[echo "Number 1: " $1 +];
[echo "Number 2: " $2 +];
