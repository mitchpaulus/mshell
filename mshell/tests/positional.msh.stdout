Number 1: Hello
Number 2: World
